setup-service

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Define variables
+SERVICE_NAME="socialpredict"
+SERVICE_FILE="/etc/systemd/system/$SERVICE_NAME.service"
+WORKING_DIRECTORY="./"
+SCRIPT_PATH="$WORKING_DIRECTORY/compose-prod"
+
+# Check if running as root
+if [ "$(id -u)" -ne 0 ]; then
+    echo "This script must be run as root" 1>&2
+    exit 1
+fi
+
+# Create systemd service file
+echo "Creating systemd service file at $SERVICE_FILE"
+cat <<EOF > $SERVICE_PROJECT_FILE
+[Unit]
+Description=Docker Compose Service for $SERVICE_NAME
+Requires=docker.service
+After=docker.service
+
+[Service]
+Type=simple
+WorkingDirectory=$WORKING_DIRECTORY
+ExecStart=$SCRIPT_PATH up -d
+ExecStop=$SCRIPT_PATH down
```

Comment:
When we run this, we get:

```
root@breirfoxforecast-alpha:/home/testdeploy/socialpredict# ./SocialPredict down
### Searching for .env file ...
.env file found.
[+] Running 8/8
 ✔ Container scripts-certbot-1                 Removed                                                                                                                                                         0.8s
 ✔ Container socialpredict-nginx-container     Removed                                                                                                                                                         1.1s
 ✔ Container socialpredict-frontend-container  Removed                                                                                                                                                        10.3s
 ✔ Container socialpredict-backend-container   Removed                                                                                                                                                        10.6s
 ✔ Container socialpredict-postgres-container  Removed                                                                                                                                                         0.4s
 ✔ Network scripts_default                     Removed                                                                                                                                                         0.1s
 ✔ Network scripts_frontend_network            Removed                                                                                                                                                         0.3s
 ✔ Network scripts_database_network            Removed
```

So basically, the database is completely removed, meaning that there is no backup.

Perhaps should we create a function which warns the user of this, e.g.:

```
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
WARNING! This command will delete the database.
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Back up your postgres data before running this command if you wish to save it.
Are you sure you with to proceed? yes/No?

```

Reply:
Running ./SocialPredict down doesn't remove the database. The database files are stored in the directory ./data/postgres and are retained between up and down of the containers. And when you start the containers again, the database automatically mounts the folder ./data/postgres and uses this database. The only way to completely destroy the database is if you delete the folder ./data/postgres